wiki/en/en-Tips-Tricks-More.md

```diff
@@ -168,25 +168,25 @@ that were set by applications with the Policy-based QoS layer.
 
 In order to enable Quality of Service you must follow these instructions:
 
-In Search box beside Start menu Type: Local Group Policy Editor (enter)<br>
-In new window, (click) on the menu icon to display the Actions third panel<br>
-Looking at the first panel of the Local Group Policy Editor<br>
-&nbsp;Local Computer Policy<br>
-&nbsp;&nbsp;Computer Configuration<br>
-&nbsp;&nbsp;&nbsp;Windows Settings<br>
-&nbsp;&nbsp;&nbsp;&nbsp;Policy-based QoS (click)<br>
-Looking at the third panel (Actions) of the Local Group Policy Editor<br>
-&nbsp;Policy-based QoS<br>
-&nbsp;&nbsp;More Actions<br>
-&nbsp;&nbsp;&nbsp;Create new Policy (click)<br>
-&nbsp;&nbsp;&nbsp;&nbsp;Policy Name: Jamulus<br>
-&nbsp;&nbsp;&nbsp;&nbsp;Specify DSCP value: 32<br>
-&nbsp;&nbsp;&nbsp;&nbsp;Next<br>
-&nbsp;&nbsp;&nbsp;&nbsp;This QoS policy applies Only to applications with name Jamulus.exe<br>
-&nbsp;&nbsp;&nbsp;&nbsp;Next<br>
-&nbsp;&nbsp;&nbsp;&nbsp;Next<br>
-&nbsp;&nbsp;&nbsp;&nbsp;UDP<br>
-&nbsp;&nbsp;&nbsp;&nbsp;Finish<br>
-(Notice Jamulus policy in center panel may be edited)<br>
+In Search box beside Start menu Type: Local Group Policy Editor (enter)
```

Comment:
PR title needs updating "Update Linux client/server for binary installs"..??!

Reply:
Not sure what you mean.